packages/angular/src/app/pages/planning-task-list/task-list-gantt/task-list-gantt.component.html

```diff
@@ -0,0 +1,44 @@
+<dx-gantt *ngIf="!isLoading" taskListWidth="500" scaleType="weeks" height="700">
```

Comment:
For property numbers use square brackets( [height]="700" )

Reply:
Fixed